examples/framework-multiple/src/pages/index.astro

```diff
@@ -32,6 +34,7 @@ import SvelteCounter from '../components/SvelteCounter.svelte';
             margin-bottom: 2em;
         }
     </style>
+    <HydrationScript />
```

Comment:
I’m pretty sure this is unnecessary.

Reply:
Turns out it is! I was getting an error before with whatever method I was using.